### PR TITLE
Merge develop into master

### PR DESCRIPTION
**Pull request by imtmcdonald**
## Description

- Configured for deployment to fly.io
- Updated dependencies to make the app functional
- Mitigated high vuln

## Related Issue

https://gohypergiant.atlassian.net/browse/OPS-324

## Motivation and Context

Users requested a glossary bot for Slack

## How Has This Been Tested?

Deployed to fly.io and validated via Slack.

## Screenshots (if appropriate):
image

### CODE REVIEW

**Comment by imtmcdonald** [comment]
Note, the app is currently spun down in fly.io to save on resources. Once I get the go ahead, I'll spin up.

File: .github/workflows/set-fly-app-name.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    inputs:
+      ref:
+        type: string
+        description: "The ref to format"
+        required: true
+      prefix:
+        type: string
+        description: "The prefix to add to the name"
+        required: false
+        default: "hg-cc-eph-"
```
Comment (bbriggs) [inline .github/workflows/set-fly-app-name.yaml]: Change prefix default (or drop it altogether).

File: .github/workflows/deploy-to-fly.yaml [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+          --name ${{ inputs.app_name }} \
+          --now \
+          --org=${{ vars.FLY_ORG }} \
+          --region=${{ vars.FLY_REGION }} \
+          --remote-only \
+          --reuse-app \
+          $(printf "%s" "$ENV_PARAMS" | sed 's/^/-e /') \
+          -e FLY_APP_NAME=${{ inputs.app_name }} \
+          -e SLACK_TOKEN=${{ secrets.GLOSS_BOT_SLACK_TOKEN }} \
+          -e SLACK_WEBHOOK_URL=${{ vars.SLACK_WEBHOOK_URL }} \
```
Comment (bbriggs) [inline .github/workflows/deploy-to-fly.yaml]: Slack webhooks are sensitive. Make this a secret.

**Review by bbriggs — CHANGES_REQUESTED** [review]
Two minor changes. Comments inline.

Also, we might want to consider abstracting the fly deployment workflow now that we're using it across multiple repos. Work for a later PR.

**Review by bbriggs — APPROVED** [review]

**Pull request merged**